common/src/buffers/in_mem/send.rs

```diff
@@ -25,6 +28,20 @@ pub struct InMemorySendingBuffer {
     buffer: Arc<Mutex<Buffer>>,
 }
 
+#[derive(Clone, bon::Builder)]
+pub struct InMemorySendingBufferConfig {
+    min_payload_length: u8,
+    q: f32,
+}
+
+#[async_trait]
+impl SendingBufferConfig for InMemorySendingBufferConfig {
+    type Buffer = InMemorySendingBuffer;
+    async fn create(&self) -> Result<InMemorySendingBuffer, DenimBufferError> {
+        InMemorySendingBuffer::new(self.q, self.min_payload_length)
+    }
+}
```

Comment:
same here

Reply:
this will be done when i make e2e in my next pull request